src/app/home-page/img/background.svg

```diff
@@ -0,0 +1 @@
#2f99b8" stroke="#2f99b8" stroke-width="1.51"/><path d="M3444,-72L3449,-219L3306,-218Z" fill="#2f7497" stroke="#2f7497" stroke-width="1.51"/><path d="M3617,229L3613,68L3458,73Z" fill="#30779a" stroke="#30779a" stroke-width="1.51"/><path d="M3764,94L3613,68L3617,229Z" fill="#2f7295" stroke="#2f7295" stroke-width="1.51"/><path d="M694,2065L856,2082L845,1924Z" fill="#2f94b4" stroke="#2f94b4" stroke-width="1.51"/><path d="M1137,2221L1158,2384L1299,2246Z" fill="#348baf" stroke="#348baf" stroke-width="1.51"/><path d="M3760,534L3773,396L3621,400Z" fill="#2f8cad" stroke="#2f8cad" stroke-width="1.51"/><path d="M3918,388L3773,396L3760,534Z" fill="#3087a8" stroke="#3087a8" stroke-width="1.51"/><path d="M3299,2245L3287,2068L3154,2245Z" fill="#2f7c9f" stroke="#2f7c9f" stroke-width="1.51"/><path d="M3299,2245L3441,2070L3287,2068Z" fill="#307a9d" stroke="#307a9d" stroke-width="1.51"/><path d="M856,2082L1003,2245L1006,2063Z" fill="#2f92b2" stroke="#2f92b2" stroke-width="1.51"/><path d="M543,1455L552,1631L683,1608Z" fill="#2faecc" stroke="#2faecc" stroke-width="1.51"/><path d="M680,1765L700,1922L857,1770Z" fill="#2da5c3" stroke="#2da5c3" stroke-width="1.51"/><path d="M396,1467L552,1631L543,1455Z" fill="#31a7c8" stroke="#31a7c8" stroke-width="1.51"/><path d="M683,-240L852,-228L1307,-231Z" fill="#308faf" stroke="#308faf" stroke-width="1.51"/><path d="M1307,-231L852,-228L1165,-215Z" fill="#338daf" stroke="#338daf" stroke-width="1.51"/><path d="M992,-213L852,-228L859,-70Z" fill="#2f8cad" stroke="#2f8cad" stroke-width="1.51"/><path d="M856,2082L843,2224L1003,2245Z" fill="#2f8cad" stroke="#2f8cad" stroke-width="1.51"/><path d="M528,1927L700,1922L680,1765Z" fill="#2f9ab9" stroke="#2f9ab9" stroke-width="1.51"/><path d="M523,1157L375,1306L536,1312Z" fill="#3b9cc5" stroke="#3b9cc5" stroke-width="1.51"/><path d="M387,843L538,687L374,694Z" fill="#369fc4" stroke="#369fc4" stroke-width="1.51"/><path d="M1003,2245L1158,2384L1137,2221Z" fill="#328daf" stroke="#328daf" stroke-width="1.51"/><path d="M1320,2386L1002,2394L1608,2398Z" fill="#3688af" stroke="#3688af" stroke-width="1.51"/><path d="M3289,2377L3299,2245L3154,2245Z" fill="#307c9f" stroke="#307c9f" stroke-width="1.51"/><path d="M3595,-85L3449,-219L3444,-72Z" fill="#2f6f93" stroke="#2f6f93" stroke-width="1.51"/><path d="M3306,-218L3604,-231L3136,-224Z" fill="#2f779a" stroke="#2f779a" stroke-width="1.51"/><path d="M3136,-224L3604,-231L2699,-237Z" fill="#3081a3" stroke="#3081a3" stroke-width="1.51"/><path d="M374,694L538,687L530,553Z" fill="#31a6c7" stroke="#31a6c7" stroke-width="1.51"/><path d="M381,537L374,694L530,553Z" fill="#30a5c4" stroke="#30a5c4" stroke-width="1.51"/><path d="M528,79L524,244L705,219Z" fill="#308cad" stroke="#308cad" stroke-width="1.51"/><path d="M528,79L705,219L681,77Z" fill="#308aab" stroke="#308aab" stroke-width="1.51"/><path d="M398,1006L386,1143L523,1157Z" fill="#4094c2" stroke="#4094c2" stroke-width="1.51"/><path d="M241,1009L386,1143L398,1006Z" fill="#3f8fbe" stroke="#3f8fbe" stroke-width="1.51"/
```

Comment:
All assets/resources like images should be on a folder on the project root folder